test/integration/test_expid.py

```diff
@@ -92,16 +111,16 @@ def test_expid_mutually_exclusive_arguments(type_flag: str, autosubmit: Autosubm
 
 
 @pytest.mark.parametrize(
-        'has_min_yaml',
-        [
-            True,
-            False
-        ]
+    'has_min_yaml',
+    [
+        True,
+        False
+    ]
 )
 def test_copy_minimal(has_min_yaml: bool, autosubmit: Autosubmit) -> None:
     """
     Test for issue 2280, ensure autousbmit expid -min --copy expid_id cannot be used if the experiment
```

Comment:
Type here if you'd like to remove one more warning :grimacing: s/autousbmit/autosubmit